src/main/resources/static/api.yml

```diff
@@ -997,6 +1032,154 @@ paths:
+      responses:
+        200:
+          description: successfully retrieved assessment
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Assessment'
+        401:
+          $ref: '#/components/responses/401Response'
+        403:
+          $ref: '#/components/responses/403Response'
+        500:
+          $ref: '#/components/responses/500Response'
+    put:
+      tags:
+        - Assessment data
+      summary: Updates an assessment
+      parameters:
+        - in: path
+          name: assessmentId
+          required: true
+          description: Id of the assessment
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        description: Updated assessment
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/UpdateAssessment'
+        required: true
+      responses:
+        200:
+          description: successfully retrieved assessment
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Assessment'
+        401:
+          $ref: '#/components/responses/401Response'
+        403:
+          $ref: '#/components/responses/403Response'
+        500:
+          $ref: '#/components/responses/500Response'
+  /assessments/{assessmentId}/notes:
+    post:
+      tags:
+        - Assessment data
+      summary: Adds a clarification note to an assessment
+      parameters:
+        - in: path
+          name: assessmentId
+          required: true
+          description: Id of the assessment
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        description: Clarification note
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/NewClarificationNote'
+        required: true
+      responses:
+        201:
+          description: successfully created a clarification note
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/ClarificationNote'
+        401:
+          $ref: '#/components/responses/401Response'
+        403:
+          $ref: '#/components/responses/403Response'
+        500:
+          $ref: '#/components/responses/500Response'
+  /roles:
+    get:
+      tags:
+        - Auth
+      summary: Lists approved premises roles for the logged in user
+      responses:
+        200:
+          description: successfully retrieved roles
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  type: string
+                  enum:
+                    - AP_ASSESSOR
+                    - AP_MATCHER
+                    - AP_MANAGER
+                    - AP_WORKFLOW_MANAGER
+                    - AP_APPLICANT
+                    - AP_ROLE_ADMIN
```

Comment:
assuming that our service manages these roles (they aren't in the JWT) do you think it's necessary to have the `AP_*` prefix? Would it have value in system wide logging or auditing do you think? Or would it just be cruft?

Reply:
Yeah they do seem redundant, will remove the prefix